lib/editor/components/ExceptionDateRange.js

```diff
@@ -76,84 +91,82 @@ export default class ExceptionDateRange extends Component<Props> {
   }
 
   render () {
-    const {range, index, isInvalid} = this.props
+    const {index, range, validationState} = this.props
     const startMoment = range.startDate ? moment(range.startDate) : undefined
 
     const dateTimeProps = {
       defaultText: undefined,
       mode: 'date',
       size: 'sm',
-      style: {
-        gridColumn: 'span 5'
-      }
+      style: {gridColumn: 'span 5'}
     }
 
-    // Highlight both inputs red if we have invalid input.
-    const rangeInputStyleProps = {...inputStyleProps, backgroundColor: isInvalid ? '#FBDEDE' : undefined}
-
     return (
-      <div
-        key={index}
-        style={{
-          display: 'grid',
-          gridGap: '5px',
-          gridTemplateColumns: 'repeat(12, 1fr)',
-          marginBottom: '5px'
-        }}
-      >
+      // Validation state override allows for control of input before date
+      <FormGroup validationState={validationState === 'error' || this.state.validationStateOverride ? 'error' : 'success'}>
         <div
```

Comment:
Same as other comment, we could merge this `<div>` with the `<FormGroup>` above.

Reply:
Same as above